docs/docs/routing.md

```diff
@@ -0,0 +1,241 @@
+---
+id: routing
+title: Routing
+permalink: docs/routing.html
+---
+
+Routing is the process of choosing what to render based on the browser's current URL.
+
+The simplest way to implement Routing in a React application is with a top-level Component that chooses which content to render by URL. For example, a top-level `App` component may decide what to render based on [window.location.pathname](https://developer.mozilla.org/en-US/docs/Web/API/Location):
+
+```js
+function Menu() {
+  return (
+    <div>
+      <a href="/news">News</a>
+      <a href="/profile">Profile</a>
+    </div>
+  );
+}
+
+class App extends React.Component {
```

Comment:
Why is this a class? Since it doesn't use lifecycle or state, let's use a functional component.